packages/cozy-doctypes/src/File.js

```diff
@@ -23,6 +23,52 @@ class CozyFile extends Document {
     const parentDirectoryPath = trimEnd(parentDir.path, '/')
     return `${parentDirectoryPath}/${name}`
   }
+
+  /**
+   * Move file to destination.
+   *
+   * @param   {string} fileId               - The file's id (required)
+   * @param   {object} destination
+   * @param   {string} destination.folderId - The destination folder's id (required)
+   * @param   {string} destination.path     - The file's path after the move (optional)
```

Comment:
I don't think it is optional no? It is used and required only in case of `force` === true. The comment should be explicit about why we need it.